articles/identity-labs/01-web-sign-in/exercise-01.md

```diff
@@ -28,14 +28,17 @@ A simple Node.js Express application has been created to get you started. This i
 
 The following steps will show you how to set up and run the web application that you will be securing:
 
-1. Go to the `/lab-01/begin` folder in your local copy of the [identity exercise repo](https://github.com/auth0/identity-102-exercises/).
+1. Open your Terminal app, clone the [identity exercise repo](https://github.com/auth0/identity-102-exercises/), then go to the `/lab-01/begin` folder:
 
-::: note
-If you've never cloned a repo before, please see [GitHub's instructions ](https://help.github.com/en/articles/cloning-a-repository).
-:::
+```bash
```

Comment:
Much better to tell them what to do in a lab than send them off to generic instructions :) Nice improvement.

Reply:
I agree :) this is always a catching point for a number of folks and it only occurred to me know that it could be way more clear, particularly for folks who have never used the command line.